tests/regression/test_bcs.py

```diff
@@ -1,403 +0,0 @@
-import pytest
```

Comment:
Well, this file was definitely testing BC functionality (e.g. bc.apply and so forth).  Moreover, you didn't rename it, because all of the tests in this file are now gone (and to reappear in test_to_expression.py).